roles/pool/config-examples/pool-config-hosted-tp-example.toml

```
@@ -29,4 +29,4 @@ pool_signature = "Stratum V2 SRI Pool"
 tp_address = "75.119.150.111:8442"
 tp_authority_public_key = "9bwHCYnjhbHm4AS3pWg9MtAH83mzWohoJJJDELYBqZhDNqszDLc"
 shares_per_minute = 1.0
```

let's take the opportunity of this PR and increase this value

1 share per minute is too low for most testing scenarios
```suggestion
shares_per_minute = 1.0
```

```suggestion
shares_per_minute = 6.0
```